release-notes.md

```diff
@@ -1,5 +1,70 @@
 # Release Notes
 
+## 3.0
+
+medic-conf v3.0 contains breaking changes! This release only impacts the `compile-app-settings` action and the partner code written in `tasks.js`, `targets.js`, `contact-summary.js`, and `contact-summary.templated.js`.
+
+### Breaking Change - Static Analysis via EsLint
+medic-conf has changed the engine we use for performing static analysis from [JsHint](https://jshint.com/) to [eslint](https://eslint.org/). This grants authors more control over the static analysis rules which are enforced on their code. We provide a set of default rules, but you can create an [.eslintrc file](https://eslint.org/docs/user-guide/configuring#using-configuration-files-1) to override these defaults or add rules. The default static analysis rules which are enforced in medic-conf 3.0 are similar to the rules which were enforced in medic-conf 2.x, but details may be slightly different.
+
+### Support for ECMAScript 6
+In previous versions of medic-conf, Javascript was limited to ES5. [ECMAScript 6](http://es6-features.org/) was a huge change for the JavaScript language and community. To use ES6 in your JavaScript configuration code, add a `.eslintrc` file to your project directory with the following contents:
```

Comment:
May be nice to have a small statement on why ES6 was a huge change for the JS language and community, while still allowing for pre-ES6 code.

Reply:
Instead, I just went with something a bit less opinionated. 

[ECMAScript 6](http://es6-features.org/) is the next generation of the JavaScript language and is what most modern JavaScript developers are used to.

Reply:
I am not seeing your changes here either. Did you push any?